analysis/measure_definition.py

```diff
@@ -0,0 +1,71 @@
+import argparse
+from datetime import date
+
+from ehrql import INTERVAL, Measures, codelist_from_csv, months
+from ehrql.tables.core import clinical_events as events
+from ehrql.tables.core import patients
+from ehrql.tables.tpp import practice_registrations as registrations
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--codelist")
+args = parser.parse_args()
+
+start_date = date(2024, 4, 1)
+
+
+def num_months(from_, to_):
+    """Returns the number of months between the given dates."""
+    return abs(to_.year - from_.year) * 12 + abs(from_.month - to_.month)
+
+
+# same year, same month
+assert num_months(date(2025, 1, 1), date(2025, 1, 31)) == 0
+# different year, different month
+assert num_months(date(2024, 1, 1), date(2025, 2, 1)) == 13
+# to_ before from_
+assert num_months(date(2025, 2, 1), date(2024, 1, 1)) == 13
```

Comment:
I realised that there was a bug in `num_months`. I've fixed it, and have added some lightweight tests: namely, the three `assert`s here. I'd like to highlight that they're _lightweight_ tests. One day, we may choose to move them to test functions, which we may choose to call with, for example, [pytest](https://docs.pytest.org/en/stable/). Not today, however 🙂